diff --git a/Doc/tools/static/version_switch.js b/Doc/tools/static/version_switch.js
index 7289a4d..c3cc107 100644
--- a/Doc/tools/static/version_switch.js
+++ b/Doc/tools/static/version_switch.js
@@ -27,8 +27,8 @@
   }

   function patch_url(url, new_version) {
-    var url_re = /\.org\/(\d|py3k|dev|((release\/)?\d\.\d[\w\d\.]*))\//,
-        new_url = url.replace(url_re, '.org/' + new_version + '/');
+    var url_re = /\/(\d|py3k|dev|((release\/)?\d\.\d[\w\d\.]*))\//,
+        new_url = url.replace(url_re, '/' + new_version + '/');

     if (new_url == url && !new_url.match(url_re)) {
       // python 2 url without version?
